<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addnewstudent</class>
 <widget class="QDialog" name="addnewstudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lastNameEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="studentIDEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="genderBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="socialIDEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Social ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Birth date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthdateEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
